docs/source/tutorial/tutorial-execute-query.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "3. Execute your first query"
+---
+
+## Constructing a query in GraphiQL
+
+The most common GraphQL operation is the **query**, which requests data from your graph in a structure that conforms to your server's schema. If you return to [the GraphiQL query explorer](https://n1kqy.sse.codesandbox.io/) for your server, you can see available queries in the Schema tab you opened earlier. 
+
+Click on the `launches` query at the top for details about it:
+
+![detail about launches query](images/launches_detail.png)
+
+In the right panel, you see both the query itself and information about what the query returns. You can use this information to write a query you'll eventually add to your app. 
+
+In the left-hand text area, add the following lines to start creating a query that will fetch a list of all available launches: 
+
+```graphql
+query LaunchList {
+}
+```
+
+The Apollo iOS SDK requires every query to have a name (even though this isn't required by the GraphQL spec). The query above has the name `LaunchList`.
+
+Next, between the query's curly braces, start typing `la`. An autocomplete box pops up and shows you options based on what's in the schema:
+
+![example of autocomplete](images/grapqhiql_autocomplete.png)
+
+GraphiQL is a great tool for building and verifying queries so you don't have to repeatedly rebuild your project in Xcode to try out changes. 
+
+As the schema indicates, the `launches` query returns a `LaunchConnection` object that includes both a list of launches and fields related to pagination (`cursor` and `hasMore`). The query you write indicates exactly which fields of the `LaunchConnection` object you want to be returned, like so:
+
+```graphql
+query LaunchList {
+  launches {
+    cursor
+    hasMore
+  }  
+}
+```
+
+If you run this query by pressing the play button in GraphiQL, the query returns results as a JSON object on the right-hand side of the page: 
+
+![basic query JSON in GraphiQL](images/completed_basic_query.png)
+
+This query executes successfully, but it doesn't include any information about the `launches`! That's because we didn't include the necessary field in our query.
```

Comment:
I think it's worth keeping the explanation of the distinction between `Launch` and `LaunchConnection` here - it explains why you hit the `launches` endpoint but didn't get an array of launches back as the top-level object